src/components/bookList/BookList.css

```diff
@@ -1,52 +1,106 @@
 .bookList__message {
   text-align: center;
-  color: #fff;
+  color: var(--black);
   margin-top: 10px;
 }
 
-.bookCard__section {
-  margin-top: 20px;
-}
-
-.bookCard__section__container {
+.bookCard__container {
   max-width: 800px;
   margin: 0 auto;
   display: flex;
-  justify-content: center;
-  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin-top: 20px;
+  margin-bottom: 20px;
   gap: 10px;
+  width: 100%;
+  background-color: var(--white);
+  padding: 8px;
+  border: 1px solid var(--border);
 }
```

Comment:
- [ ] Great job so far styling your cards. However, looking at the original design given in the project requirements carefully you would notice that the `padding` you have at the `top` and `bottom` of your is a bit smaller as compared to the original design so kindly add some for padding to it to match that of the design given.

- [ ] Kindly consider increasing your `max-width` as it's a bit smaller compared to that of the original design. I would suggest you use `80vw` as that might help but yes play around with the width and know what best matches the original design. See below for more details:

- [ ] Also, Your design is missing the `progress` bar on the card. Kindly consider adding it so your design matches that of the original.
<table>
<tr>
<th align=“middle”>Original design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="864" alt="Screenshot 2023-08-24 at 17 14 24" src="https://github.com/Felipe-Perez-Ferraro/bookstore/assets/107192076/fb1f90ee-0eb6-4a42-96c3-9c2d03b440e8">
</td>
<td>
<img width="1440" alt="Screenshot 2023-08-24 at 17 12 03" src="https://github.com/Felipe-Perez-Ferraro/bookstore/assets/107192076/d5857423-bbaa-49b8-bc68-9e485034ed95">
</td>
</tr>
</table>